docs/conf.py

```diff
@@ -42,4 +42,4 @@
 
 # API links extension, stolen from Twisted's Sphinx setup
 extensions.append('apilinks')
-apilinks_base_url = 'https://twistedmatrix.com/documents/current/api/'
\ No newline at end of file
+apilinks_base_url = 'https://twistedmatrix.com/documents/current/api/'
```

Comment:
Should we have a lint that warns on files not ending in a newline?

Reply:
*shrug*